core/src/main/resources/jenkins/security/ApiTokenProperty/config.groovy

```diff
@@ -1,36 +0,0 @@
-/*
- * The MIT License
- *
- * Copyright (c) 2011, CloudBees, Inc.
- *
```

Comment:
Ensure there is no inheriting classes using it somehow.
It would be strange if it happens, but who knows...

Reply:
By replacing the groovy by a jelly, there is no problem no ?